src/bicubic_interp_mod.F90

```diff
@@ -1,10 +1,10 @@
 !> @file
-!! @brief Bicubic interpolation routines for scalars and vectors
-!! @author Mark Iredell, Kyle Gerheiser
+!> @brief Bicubic interpolation routines for scalars and vectors
+!> @author Mark Iredell, Kyle Gerheiser
 
-!> Bicubic interpolation routines for scalars and vectors
-!!
-!! @author George Gayno, Mark Iredell, Kyle Gerheiser
+!> @brief Bicubic interpolation routines for scalars and vectors.
```

Comment:
One is for the file, the other for the module.

Reply:
Is the second @brief for the 'module' statement only?

Reply:
Yes. One brief for the file, one for the module. Then, when doxygen lists the file we see it:

![image](https://user-images.githubusercontent.com/38856240/176484458-145b1610-f502-48e0-9ad0-c4b684fd5eb1.png)

And when we look at the list of modules we also see it:

![image](https://user-images.githubusercontent.com/38856240/176484671-aadd8a78-f287-4337-afb2-a7a15ae1af55.png)

This seems redundant but if we were to write a file with two modules, it would work out. Also, if we don't use both briefs, either the file list or the module list would have no documentation.

Reply:
Great.